tools-local/init-build.proj

```diff
@@ -26,7 +26,10 @@
   </Target>
 
   <Target Name="BuildTasks">
-    <Exec Command="$(DotNetCliToolDir)dotnet build tasks\Microsoft.DotNet.SourceBuild.Tasks\Microsoft.DotNet.SourceBuild.Tasks.csproj" />
+    <PropertyGroup Condition="'$(OfflineBuild)' == 'true'">
+      <OfflineSources>$(PrebuiltPackagesPath)</OfflineSources>
+    </PropertyGroup>
+    <Exec Command="$(DotNetCliToolDir)dotnet build tasks\Microsoft.DotNet.SourceBuild.Tasks\Microsoft.DotNet.SourceBuild.Tasks.csproj /p:RestoreSources=$(OfflineSources)" />
```

Comment:
Does this work online?  I'd expect blank RestoreSources to cause issues but if that works I'm fine with it.

Reply:
This does work online.  blank causes it to use the default.